frameworks/person-escort-record/questions/child-date-of-birth.yml

```diff
@@ -0,0 +1,7 @@
+type: text
+question: Child's date of birth
```

Comment:
Can we use a quotation mark instead of a prime. It's something I always notice now since someone pointed [the difference](https://carsonparkdesign.com/quotation-marks-apostrophes-versus-primes/#:~:text=Single%20primes%20are%20used%20to,and%20double%20primes%20for%20inches.&text=The%20apostrophe%2C%20or%20single%20closed,used%20for%20contractions%20or%20possessives.) out to me :)